software/firmware/.gitignore

```diff
@@ -0,0 +1,6 @@
+# IDE
+.idea/
```

Comment:
Not sure, if it makes sense to put your IDE files here, as we do not force or suggest any specific IDE. This would better be solved on your side with a global gitignore.

Reply:
is plausible and fixed